flink-formats/flink-sequence-file/src/test/java/org/apache/flink/formats/sequencefile/SerializableHadoopConfigurationTest.java

```diff
@@ -55,8 +57,8 @@ public void customPropertiesSurviveSerializationDeserialization()
         final SerializableHadoopConfiguration deserializableConfigUnderTest =
                 deserializeAndGetConfiguration(serializedConfigUnderTest);
 
-        Assert.assertThat(
-                deserializableConfigUnderTest.get(), hasTheSamePropertiesAs(configuration));
+        assertThat(deserializableConfigUnderTest.get())
+                .satisfies(matching(hasTheSamePropertiesAs(configuration)));
```

Comment:
Same question about the potential custom assertion.

Reply:
As discussed offline, we decided to keep it as is because converting the Matchers into AssertJ Assertion classes introduces a lot of boilerplate and feels very clunky